GoTrue/src/commonMain/kotlin/io/github/jan/supabase/gotrue/providers/builtin/Email.kt

```diff
@@ -39,6 +41,7 @@ data object Email : DefaultAuthProvider<Email.Config, Email.Result> {
     data class Result(
         val id: String,
         val email: String,
+        val identities: List<Identity>?,
```

Comment:
```suggestion
        val identities: List<Identity>? = null,
```
Just in case, if for some reason no identities entry is there at all.